prdoc/pr_6454.prdoc

```diff
@@ -0,0 +1,32 @@
+title: 'rpc server: fix host filter for localhost'
+doc:
+- audience: Node Operator
+  description: "So, this PR fixes an issue that I discovered using cURL where it tries\
```

Comment:
I think this is a bit of a weird format, simply having a description on separate lines doesn't pass the CI check?

```
description: |
      This PR implements the `archive_unstable_storageDiff` rpc-v2 method.
      ... add line here?
```